extensions/resource-deployment/src/ui/modelViewUtils.ts

```diff
@@ -429,7 +429,7 @@ export function setModelValues(inputComponents: InputComponents, model: Model):
 				value = inputValue.name;
 			}
 		} else {
-			throw new Error(`Unknown input type with ID ${input.id}`);
+			throw new Error(`${localize('modelViewUtils.setModelValues.Unknown.input.id', 'Unknown input type with ID')} ${input.id}`);
```

Comment:
loc team won't be able to localize this.  this comment applies to other places as well. 
1. if you are throwing error especially the ones should only be seen at dev time, you don't have to localize them. 
2. if localization is really needed, you should do localize('key', "blah blah {0} blah blah {1}", arg0, arg1), also the actual string needs to be double quoted.

Reply:
reverted.

Reply:
reverted